server/insights/src/main/java/org/infinispan/server/insights/logging/Log.java

```diff
@@ -43,17 +40,21 @@ public interface Log extends BasicLogger {
    @Message(value = "Remote Insights reporting", id = 32005)
    void insightsEnabled();
 
-   @Message(value = "Error configuring Red Hat Insight client", id = 32006)
-   CacheConfigurationException insightsConfigurationError();
+   @LogMessage(level = INFO)
+   @Message(value = "The environment variable %s or the system property %s cannot be set to true in order to " +
```

Comment:
The message seems to be unnecessarily complicated. Personally I'd go with something simpler like: `... {OPT_OUT} set to true, the Insights reports will be disabled....`